addons/t9n/models/resource.py

```diff
@@ -1,11 +1,16 @@
-from odoo import fields, models
+from odoo import fields, models, api, _, Command
+from odoo.exceptions import ValidationError
+import polib
+import base64
 
 
 class Resource(models.Model):
     _name = "t9n.resource"
     _description = "Resource file"
 
-    name = fields.Char("Resource")
+    file_name = fields.Char()
+    file = fields.Binary("Upload Resource File",
+                         required=True, attachment=False)
```

Comment:
I don't think we want to store the file. Rather, we want to send it to the Python side, read it, populate the database with its contents, and then get rid of it